health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/UpdateChecklist.js

```
@@ -388,53 +388,56 @@ const UpdateChecklist = () => {
-            let checklistTypeTemp = checklistType.toUpperCase().replace(/ /g, "_");
-            if (checklistTypeCode) checklistTypeTemp = checklistTypeCode;
-            let roleTemp = role.toUpperCase().replace(/ /g, "_");
-            uniqueLocal.push({
-                code: `${campaignName}.${checklistTypeTemp}.${roleTemp}`,
-                locale: locale,
-                message: `${t(checklistTypeLocal)} ${t(roleLocal)}`,
-                module: "hcm-checklist"
+          // Prepare localization data
+          let checklistTypeTemp = checklistType.toUpperCase().replace(/ /g, "_");
+          if (checklistTypeCode) checklistTypeTemp = checklistTypeCode;
+          let roleTemp = role.toUpperCase().replace(/ /g, "_");
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'roleTemp' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)